contracts/mediator/Initializable.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity 0.4.24;
+
+import "../upgradeability/EternalStorage.sol";
+
+contract Initializable is EternalStorage {
+    bytes32 internal constant INITIALIZED = keccak256(abi.encodePacked("isInitialized"));
+    bytes32 internal constant DEPLOYED_AT_BLOCK = keccak256(abi.encodePacked("deployedAtBlock"));
```

Comment:
don't think that we need to keep a block where the contracts was deployed for the mediators. If the same is in the repo `poa-bridge-contracts` consider to create a PR there as well.

Reply:
Removed setting the block on initialization https://github.com/poanetwork/cryptokitties-xdai-demo/pull/6/commits/e7f7b39edece2a8f4f7a089cb73b812c1cddd194. Since now we use `Initializable` from `poa-bridge-contracts` a PR will be open to fix it there for other erc677 mediator too,  and later submodule will be updated.

Reply:
Thanks! I have opened https://github.com/poanetwork/poa-bridge-contracts/issues/291.